doc/cug.tex

```diff
@@ -4551,6 +4489,380 @@ \subsubsection{Collapsible Families In Suite Graphs}
+    same workflow.]{\scriptsize Static chunked and dynamic cycled versions of
+    the same workflow.}
+    \label{fig-eg2}
+\end{figure}
+
+Dynamic cycling and static chunking can both be used in the same suite. Here's
+a yearly dynamic cycle with four static chunks within each cycle:
+
+\begin{lstlisting}
+[cylc]
+    [[parameters]]
+        chunk = 1..4
+[scheduling]
+    initial cycle point = 2020-01
+    [[dependencies]]<chunk=4>
+        [[[P1Y]]]
+            graph = """model<chunk-1> => model<chunk>
+                    model<chunk=4>[-P1Y] => model<chunk=1>"""
+\end{lstlisting}
+
+Note the inter-cycle trigger that connects the first chunk in each cycle to
+the last chunk in the previous cycle.  Of course it would be easier to use
+a three-month dynamic cycle:
+\begin{lstlisting}
+[scheduling]
+    initial cycle point = 2020-01
+    [[dependencies]]
+        [[[P3M]]]
+            graph = "model[-P3M] => model"
+\end{lstlisting}
+
+There is at least one genuine use-case for mixed static and dynamic cycling,
+however.  Consider a dynamic date-time cycling workflow of model runs that
+can take too long to run (and get killed for exceeding a wall clock limit), or
+not, depending on which of several platforms is available to run on at the
+time.  This could be handled without changing the date-time cycling structure
+of the whole suite, by splitting each run into a variable number of short
+chunks and using more chunks on the slower platform.
+
+Finally, there is one other feature of static chunking that can provide a small
+advantage.  In dynamic cycling cylc automatically ignores any dependence on
+cycle points earlier than the suite initial point (this isn't strictly
+necessary but it does simplify scheduling configuration a little).  So this,
+\begin{lstlisting}
+    graph = "model[-P1D] => model"
+\end{lstlisting}
+simplifies at the suite initial cycle point to this,
+\begin{lstlisting}
+    graph = "model"
+\end{lstlisting}
+Similarly, parameter offsets are ignored if they extend beyond the start of the
+list of parameter values.  So this,
+\begin{lstlisting}
+    graph = "model<chunk-1> => model<chunk>"
+\end{lstlisting}
+simplifies for \lstinline@chunk=1@ to this,
+\begin{lstlisting}
+    graph = "model_chunk1"
+\end{lstlisting}
+There is a parameter offset cutoff for every suite parameter list, but there is
+only one cycle point cutoff at the suite initial cycle point.  Because of this,
+if you want to replicate a workflow over several runs {\em with different start
+points} this simplification helps with all of the runs in the static case, but
+only one of them in the dynamic case - namely the one that starts at the suite
+initial cycle point.  In the future we plan to provide this kind of
+simplification at the start of any date-time sequence, not just the suite
+initial point.  It will be optional though, because delayed-start cycling tasks
+sometimes need to be triggered off earlier tasks.
```

Comment:
I guess I'm not sure what the equivalent suites are here that this applies to. From the earlier context I thought we were comparing this sort of thing:
```
[scheduling]
[[dependencies]]
[[[R/P1D]]]
  graph = model[-P1D] => model
```
to this
```
[scheduling]
[[dependencies]]
graph = model<chunk-1> => model<chunk>
```

So, I'm not clear where the gain is as the two are pretty much the same with magic to cut off the first invalid/non-existent referenced entry?